components/Slider.tsx

```diff
@@ -0,0 +1,13 @@
+export default function Slider({ children }) {
+  return (
+    <>
+      <div
+        className="scroll mb-2 flex cursor-pointer
```

Comment:
Nice, I wasn't sure if this would be possible without JS. Have you tested this on a variety of devices?

Reply:
Thanks!

I have a Mac M1 pro (MacOS 13.1) and iPhone 13 (iOS 16.2). I tested this PR on both devices, and it runs perfectly.

However, we might encounter a problem with the scroll bar on the slider with Android.

On iOS (Safari and Chrome), the scrollbar automatically hides when the user is not scrolling. But I am unsure if this behavior will persist in Android and other OSes.

Reply:
Seems to work fine on Android, scrollbar is visible and hides itself after a timeout:

![image](https://user-images.githubusercontent.com/10757768/221884035-edeceaaa-4218-4736-8885-2b372e558ac9.png)

Let's get this merged and fix any issues in a separate PR if they are reported.